### PR TITLE
00/jdix

### PR DESCRIPTION
**Pull request by jdix531**
# Requirements for Contributing a Bug Fix

- Fill out the template below. Any pull request that does not include enough
  information to be reviewed in a timely manner may be closed at the maintainers'
  discretion.

### CODE REVIEW

File: 00-dev-environment/refresh_access_token.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    print(failure_message)
+    sys.exit(1)
+
+def main():
+    aws_cred_conf_parser = configparser.ConfigParser()
+    aws_cred_conf_parser.read(AWS_CREDENTIAL_FILE_LOCATION)
+
+    if INPUT_PROFILE_NAME in aws_cred_conf_parser.sections():
+        try:
+            sts_client = boto3.client("sts", aws_access_key_id=aws_cred_conf_parser[INPUT_PROFILE_NAME]['aws_access_key_id'], aws_secret_access_key=aws_cred_conf_parser[INPUT_PROFILE_NAME]['aws_secret_access_key'])
```
Comment (jflans) [inline 00-dev-environment/refresh_access_token.py:38]: Elsewhere you (correctly) use AWS_CRED_FILE for the conf key; could be done here as well I believe. You do not need to spend any time changing this.

**Review by jflans — APPROVED** [review]
LGTM, well done.

File: 00-dev-environment/refresh_access_token.py (hunk 1 above)
Comment (jdix531) [reply 00-dev-environment/refresh_access_token.py:38]: I think this is correct unless I'm not understanding. The difference is the client is created with existing 'static' creds based on the "input" (though I left this staticly configured at the end of the day), then I store 'temp' creds named with the suffix 'temp'.

tl;dr: I use the static creds to create the temp creds. Static is 'input',  written is 'session'/'temp'. This is largely set up to handle multiple input profiles.
